packages/babel-traverse/src/path/introspection.js

```diff
@@ -158,7 +158,16 @@ export function isStatementOrBlock() {
  */
 
 export function referencesImport(moduleSource, importName) {
-  if (!this.isReferencedIdentifier()) return false;
+  if (!this.isReferencedIdentifier()) {
+    if (
+      this.isMemberExpression() &&
```

Comment:
For consistency we should also check for `OptionalMemberExpression`.

Reply:
Damn, these new kids and their fancy syntax. Seems very useless to do on a module namespace object, but you're right that we should indeed accept it :sweat_smile: And who knows, maybe someone prefers to access named imports using `?.` for asthetic reasons :joy: